lib/yardstick/rules/summary.rb

```diff
@@ -70,7 +70,7 @@ def valid?
       class SingleLine < Rule
         LINE_BREAK_CHARACTER = "\n"
```

Comment:
Not that this was part of the change, but I would probably freeze this constant.

Speaking of freezing, when something is assigned to the description I wonder if it's frozen as well. If not it probably should be.